suite-common/wallet-utils/src/__fixtures__/solanaUtils.ts

```diff
@@ -127,14 +129,52 @@ export const fixtures = {
                 data: new Uint8Array([12, 1, 0, 0, 0, 0, 0, 0, 0, 9]),
             },
         },
+        {
+            description: 'builds token transfer instruction for the Token 2022 program',
+            input: {
+                from: 'EdThAwDjfEj9joy2U7WvSsMwsHn5Wkby8R4j74qYJujz',
+                to: '4Qon5ZG7yYRkheuUwdqwN9nGRgu7oYn9tAp5tkhE77Mi',
+                owner: '6y2EP2MtSCuNE41h3gG9Fs7ZU2n24gcYiGqDpEYjDbRn',
+                amount: new BigNumber('1'),
+                mint: '8JH5uP374VW4YmVzE7LCRK9CbyRe1uXb85jK72RQzvWU',
+                decimals: 9,
+                tokenProgramName: 'spl-token-2022' as const,
+            },
+            expectedOutput: {
+                accounts: [
+                    {
+                        address: 'EdThAwDjfEj9joy2U7WvSsMwsHn5Wkby8R4j74qYJujz',
+                        role: AccountRole.WRITABLE,
+                    },
+                    {
+                        address: '8JH5uP374VW4YmVzE7LCRK9CbyRe1uXb85jK72RQzvWU',
+                        role: AccountRole.READONLY,
+                    },
+                    {
+                        address: '4Qon5ZG7yYRkheuUwdqwN9nGRgu7oYn9tAp5tkhE77Mi',
+                        role: AccountRole.WRITABLE,
+                    },
+                    expect.objectContaining({
+                        address: '6y2EP2MtSCuNE41h3gG9Fs7ZU2n24gcYiGqDpEYjDbRn',
+                        role: AccountRole.READONLY_SIGNER,
+                        signer: expect.objectContaining({
+                            address: '6y2EP2MtSCuNE41h3gG9Fs7ZU2n24gcYiGqDpEYjDbRn',
+                        }),
+                    }),
+                ],
+                data: new Uint8Array([12, 1, 0, 0, 0, 0, 0, 0, 0, 9]),
+            },
+        },
     ],
     buildCreateAssociatedTokenAccountInstruction: [
         {
-            description: 'builds create associated token account instruction',
+            description:
+                'builds create associated token account instruction for the original Token program',
```

Comment:
NP: maybe I would avoid introducing the term "original" and use "Spl token" here, it does not really matter but IMO its better to avoid adding new terminology. wdyt?

Reply:
618526f3a2491a86d2a53ab1b063d0d277e86d3e